tests/test_webapp.py

```diff
@@ -0,0 +1,40 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import unittest
+import os
+
+CHROMEDRIVER = os.getenv('CHROMEDRIVER')
+
+class TestEpiGraphHub(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.driver = webdriver.Chrome(CHROMEDRIVER)
+        cls.driver.get("https://epigraphhub.org/")
```

Comment:
I guess here it needs to be http://localhost:8088

Reply:
I'm aware, and the step should be below the containers initialization. But the chromedriver nor chrome-setup seem to be working on CI, kinda lost at this point